readme.md

```diff
@@ -1,13 +1,17 @@
 # Rackup
 
-`rackup` provides a command line interface for running a Rack-compatible application.
+`rackup` provides a command line interface for running a Rack-compatible application. It also provides a generic interface for starting a `rack`-compatible server: `Rackup::Handler`. It is not designed for production use.
 
 [![Development Status](https://github.com/rack/rackup/workflows/Test/badge.svg)](https://github.com/rack/rackup/actions?workflow=Test)
 
 ## Installation
 
 ``` bash
-$ gem install rackup
+-- For Falcon
```

Comment:
Considering relative popularity, Puma should be listed first.

Reply:
Haha, I did it by alphabetical order. Are you okay with that or do you still want me to change it?

Reply:
I recommend the change, but I approved without it, so it's up to you.

Reply:
Okay, I don't have a strong opinion, so I'll change it.